schema/me/collector_profile.js

```diff
@@ -21,6 +21,9 @@ export const CollectorProfileFields = {
   loyalty_applicant_at: date,
   professional_buyer_at: date,
   professional_buyer_applied_at: date,
+  intents: {
+    type: new GraphQLList(GraphQLString),
```

Comment:
Wondering if this is the correct type? `intents` is essentially an [array of strings](https://github.com/artsy/gravity/blob/master/app/models/domain/collector_profile.rb#L42).

Reply:
Yup, should be!